src/ethz_snow/constants.py

```diff
@@ -3,13 +3,12 @@
 # Loading yaml config file
 import pkg_resources
 import yaml
+from numpy import pi
 
 from collections.abc import Mapping
 
 from typing import Optional, List, Any
 
-from .__init__ import __citation__, __bibtex__
```

Comment:
please don't remove stuff unless you are sure that this is what you want :)

Reply:
Don't know what happened here, accidentally I guess, I put it back now.